meetings/2021-11-03.md

```diff
@@ -0,0 +1,204 @@
+
+* RV: Open question is: is this suitable for 1.0 or go for undisputed approach for 0.9.
+
+* KK: Perhaps. Would you implement it in CSS?
+
+* RV: The reason Tim wanted this.. he said you have 3 weeks to do it. Not my proudest work but it is simple. It is extensible. The support for N3 libs and support for CSS would be trivial. It is a low hanging fruit with a lot of option for extensibility. Lowest common demoniator so to speak.
+
+* KK: What do you all think?
+
+* SC: Go for it until someone
+
+* JB: ??? +1 n3 patch support
+
+* DZ: +1
+
+* KK: Doesn't it also mean we are deprecating the behaviour (in NSS)?
+
+* RV: I'd say there is disagreement around the behaviour. It is not stable yet. The stable version could be N3.
+
+* KK: At risk word would be use
+
+* DZ: What does solidOS use?
+
+* RV: SPARQL Patch. All of the code that's needed is already there.
+
+* KK: That's the client side? Right.. okay.
+
+* SC: General agreement to support N3 patch.
+
+* RV: Do I need to contribute to spec text?
+
+* KK: Yes. That'd be good.
+
+ACTION: RV to make initial draft. KK to assign relevant issues to RV. KK to record 'something' on that issue.
+
+
+### Write methods as requirements
+URL: https://github.com/solid/specification/pull/304
+
+* KK: Also for testing for compliance... if we have to test for eventuality that Solid server that does not support write operations then that makes the test suite more complex. We would want to support servers that are read-only but that is mostly just a Web server right.. At some point we want to introduce conformance classes. In addition, we want to have language that allows servers for read-only. Which is also in the spec.. Therefore to not tackle more than this then I'd like write methods for required.
+
+* RV: That clarifies a lot. I can imagine read-only servers. How do we know if a solid server announces whether htey are read or write?
+
+* KK: accept headers.
+
+* RV: or status codes.
+
+* SC: Test suite shouldn't be driving factor for the requirement
+
+* KK: I disagree with that idea because. The test suite is important for the health of the ecosystem. As a principle that is important but if there is no easy way to automate the testing then it is a toy.. you don't have a production system you have a toy.
+
+* SC: spec comes before test.. even has test modes have different ways of checking.
+
+* KK: there is principle and pragmatism. In general I agree to adopt the test to requirements. But if we go to the 
+
+* SC: then we are requiring automated test - that needs to be a specific requirement of what we expect from the requirement.
+
+* KK: what can we do that's testable in one week. instead of later on. if we want to do thta. so it does infuence a lot of stuff. I have spoken.
```

Comment:
```suggestion
* KK: what can we do that's testable in one week, instead of later on, if we want to do that? so it does influence a lot of stuff. I have spoken.
```